### ISSUE TITLE
Partner Landing Pages Not Working

### ISSUE THREAD

**Issue by jhreis** [brave/brave-ios#748]
Certain Partners have landing pages when loading from special links. These landing pages are currently not being loaded in 1.7+

Most of the code is isolated to `UserReferralProgram.swift` and `AppDelegate`

AppDelegate:

```
if isFirstLaunch {
    urp.referralLookup { url in
        guard let url = url else { return }
        postAsyncToMain(1) { try? self.browserViewController.openURLInNewTab(url.asURL()) }
    }
}
```

**Comment by jhreis** [comment]
Waiting on bizdev.

**Comment by jhreis** [comment]
Still waiting on additional information, seems that this is probably not a Brave issue.

**Comment by iccub** [comment]
Closing as stale, we can make a new ticket if the problem still persists
